### DISCUSSION TITLE
V3 modifier to skip `x-on:click` if `href="#"` or non navigation element

### DISCUSSION THREAD

**Discussion by bilogic** [livewire/livewire#6137] [1. Help]
1. I'm using the typical `x-on:click="open = !open"` to open/close a menu
2. If I were to click on a `<div ...` part of the menu (non `<a ...` or `<button ...`), the menu also closes, this behavior is quite different from most non-Livewire menus on the net
3. Most non-Livewire menus also don't close/react if it is a `<a href="#" ...`, sometimes we might want to click and select some text etc
4. Is there a modifier that can let us achieve the same behavior?

(There might be other navigation/reactive elements such as `details`/`summary`)

**Comment by joshhanley** [comment]
@bilogic can you share an example menu code? I suspect it's just a matter of where the click is being handled.
